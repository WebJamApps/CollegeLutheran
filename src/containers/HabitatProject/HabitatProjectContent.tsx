import { Grid } from '@mui/material';
import { useContext } from 'react';
import parser from 'html-react-parser';
import { ContentContext } from 'src/providers/Content.provider';
import ELCALogo from 'src/components/elcaLogo';
import PicSlider from 'src/components/PicSlider';

const HabitatGridItem = (props: { children: any; }) => {
  const { children } = props;
  return (
    <Grid item xs={12} sm={6} md={4} style={{ paddingInline: '20px' }}>
      {children}
    </Grid>
  );
};

const TeamLeaders = (props: { children: any; }) => {
  const { children } = props;
  return (
    <Grid item container direction="row">
      <div>
        <p><strong>TEAM LEADERS:</strong></p>
        {children}
      </div>
    </Grid>
  );
};

const Responsibilities = (props: { children: any; }) => {
  const { children } = props;
  return (
    <div>
      {children}
      <strong> RESPONSIBILITIES </strong>
    </div>
  );
};

const AboutProject = () => {
  const { content: { habitatPage } } = useContext(ContentContext);
  return (
    <Grid item xs={12} sm={6} md={4} style={{ marginInline: '0', paddingInline: '20px' }}>
      <h3 style={{
        marginBottom: '0px', paddingBottom: '16px', paddingTop: '16px', textAlign: 'center',
      }}
      >
        CLC Habitat Project
      </h3>
      <div style={{ textAlign: 'left' }}>
        College Lutheran Church has made a commitment of $50,000 to sponsor a home in the
        Roanoke Valley. This is the largest service project undertaken by the members of College
        Lutheran and we are delighted to be underway.
        <br />
        Here are some details:
        <section style={{ marginTop: '20px', textAlign: 'left', marginBottom: '35px' }}>
          {parser(habitatPage && habitatPage.comments ? habitatPage.comments : '')}
        </section>
        <p>
          Questions??? Contact
          {' '}
          <a href="mailto:rcrevpaul@gmail.com">rcrevpaul@gmail.com</a>
          {' '}
          or
          {' '}
          <a href="mailto:clchabitatsalem@gmail.com">clchabitatsalem@gmail.com</a>
        </p>
      </div>
    </Grid>
  );
};

const VolunteerSignUp = () => (
  <Grid item xs={12} sm={6} md={4} style={{ paddingInline: '20px' }}>
    <h4 style={{ paddingBottom: '15px', paddingTop: '22px', textAlign: 'center' }}>Volunteer Sign Ups</h4>
    <Grid item container direction="row" justifyContent="center" alignItems="center">
      <div style={{ maxWidth: '500px' }}>
        <p>
          There will be a total of 42 days for volunteering to support the habitat build.
          For each workday we need to have the following volunteers:
        </p>
        <ul>
          <li>
            10 people for construction work
          </li>
          <li>
            4 people to supply lunch/water
          </li>
          <li>
            1-2 videographers
          </li>
        </ul>
        <p>
          Please
          {' '}
          <a href="https://signup.com/client/invitation2/secure/444699447581390086/false#/invitation" target="_blank" rel="noreferrer">
            <b>click here</b>
          </a>
          {' '}
          to sign up to volunteer!
        </p>
      </div>
    </Grid>
  </Grid>
);

export const SlideShow = (): JSX.Element => {
  const { pictures } = useContext(ContentContext);
  const { habitatPics = [] } = pictures;
  return (
    <Grid item xs={12} sm={6} md={4} style={{ paddingInline: '20px' }}>
      <h4 style={{ paddingBottom: '15px', paddingTop: '22px', textAlign: 'center' }}>Progression Slideshow</h4>
      <PicSlider data={habitatPics} />
      <p style={{ textAlign: 'center' }}>
        For progression videos, please checkout our
        {' '}
        <a href="https://www.youtube.com/channel/UCOra1rXiO-BHzMDNlLd9hFQ/videos" target="_blank" rel="noopener noreferrer">
          YouTube channel
        </a>
        .
      </p>
    </Grid>
  );
};

const ConstructionInfo = () => (
  <div>
    <Responsibilities>
      <p>Matt Gart, Mike Preston, and Skip Zubrod</p>
    </Responsibilities>
    <ul>
      <li>
        Schedule workdays
      </li>
      <li>
        Schedule workers
      </li>
      <li>
        Coordinate with site supervisor
      </li>
    </ul>
  </div>
);

const SupportInfo = () => (
  <div>
    <Responsibilities>
      <p>Lisa Yagle, Kay Westermann, and Karen Zubrod</p>
      <p><strong>Sunday Updaters:</strong></p>
      <p>Roger Holtman, Anita Ungs, and Paul Brummer</p>
      <p><strong>Publicity:</strong></p>
      <p>Gary Lautenschlager</p>
      <p><strong>Web Page:</strong></p>
      <p>Josh Sherman</p>
    </Responsibilities>
    <ul>
      <li>
        Make lunches
      </li>
      <li>
        Provide water
      </li>
      <li>
        Provide safety
      </li>
    </ul>

  </div>
);

const FamilyInfo = () => (
  <div>
    <Responsibilities><p>Bob and Lu Ann Yates, Ellen Holtman, and Debi Talin</p></Responsibilities>
    <ul>
      <li>Connect with the family</li>
      <li>Provide move-in needs</li>
      <li>Introduce family to CLC</li>
    </ul>
  </div>
);

function HabitatProjectContent():JSX.Element {
  return (
    <div className="page-content">
      <Grid container>

        {/* About the Habitat Project Section */}
        <AboutProject />

        {/* Volunteer Sign Ups Section */}
        <VolunteerSignUp />

        {/* Slideshow Section */}
        <SlideShow />

        {/* Construction Team Section */}
        <HabitatGridItem>
          <h4 style={{ paddingBottom: '15px', paddingTop: '22px', textAlign: 'center' }}>Construction Team</h4>
          <TeamLeaders>
            <ConstructionInfo />
          </TeamLeaders>
        </HabitatGridItem>

        {/* Support Team Section */}
        <HabitatGridItem>
          <h4 style={{ paddingBottom: '15px', paddingTop: '22px', textAlign: 'center' }}>Support Team</h4>
          <TeamLeaders>
            <SupportInfo />
          </TeamLeaders>
        </HabitatGridItem>

        {/* Family Team Section */}
        <HabitatGridItem>
          <h4 style={{ paddingBottom: '15px', paddingTop: '22px', textAlign: 'center' }}>Family Team</h4>
          <TeamLeaders>
            <FamilyInfo />
          </TeamLeaders>
        </HabitatGridItem>

      </Grid>
      <ELCALogo />
    </div>
  );
}
export default HabitatProjectContent;
