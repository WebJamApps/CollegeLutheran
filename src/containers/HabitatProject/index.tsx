import HabitatProjectContent from './HabitatProjectContent';
import commonUtils from '../../lib/commonUtils';

const HabitatProject = (): JSX.Element => {
  commonUtils.setTitleAndScroll('Habitat Project', window.screen.width);
  return (<HabitatProjectContent />);
};
export default HabitatProject;
