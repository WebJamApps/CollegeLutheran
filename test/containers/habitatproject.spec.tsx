
import HabitatProject from '../../src/containers/HabitatProject/index';
import HabitatProjectContent from '../../src/containers/HabitatProject/HabitatProjectContent';
import React from 'react';
import renderer from 'react-test-renderer';
  
describe('Habitat Project', () => {
  it('Renders the Habitat Project page', () => {
    const page = renderer.create(<HabitatProject/>).toJSON();
    expect(page).toMatchSnapshot();
  });
  
  it('Renders DefaultHabitatContent', () => {
    const page = renderer.create(<DefaultHabitatContent/>).toJSON();
    expect(page).toMatchSnapshot();
  });
});
