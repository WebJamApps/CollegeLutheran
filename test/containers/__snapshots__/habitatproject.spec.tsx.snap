// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Habitat Project Renders DefaultHabitatContent 1`] = `
<div
  className="page-content"
>
  <div
    className="MuiGrid-root MuiGrid-container"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      style={
        {
          "marginInline": "0",
          "paddingInline": "20px",
        }
      }
    >
      <h3
        style={
          {
            "marginBottom": "0px",
            "paddingBottom": "16px",
            "paddingTop": "16px",
            "textAlign": "center",
          }
        }
      >
        CLC Habitat Project
      </h3>
      <p
        style={
          {
            "textAlign": "left",
          }
        }
      >
        College Lutheran Church has made a commitment of $50,000 to sponsor a home in the Roanoke Valley. This is the largest service project undertaken by the members of College Lutheran and we are delighted to be underway.
        <br />
        Here are some details:
        <ul>
          <li>
            This will be a rebuild project at 1004 Stewart Avenue, SE Roanoke.
          </li>
          <li>
            The existing structure will be gutted with hazardous material being mitigated before volunteers are on site.
          </li>
          <li>
            New construction will be added to the rear of the building.
          </li>
          <li>
            A family has yet to be confirmed for this home, but we expect a family will be designated in the coming weeks
          </li>
          <li>
            All level of construction skills will be needed. Workdays will probably be Wednesday and Saturday, but that may change. We typically work 8:00 â€“ 2:00, but half-time participation is welcome.
          </li>
          <li>
            Lunch, water, tools are all provided.
          </li>
        </ul>
      </p>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      style={
        {
          "paddingInline": "20px",
        }
      }
    >
      <h4
        style={
          {
            "paddingBottom": "15px",
            "paddingTop": "22px",
            "textAlign": "center",
          }
        }
      >
        Volunteer Sign Ups
      </h4>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-content-xs-center"
      >
        <div
          style={
            {
              "maxWidth": "500px",
            }
          }
        >
          <p>
            There will be a total of 42 days for volunteering to support the habitat build. For each workday we need to have the following volunteers:
          </p>
          <ul>
            <li>
              10 people for construction work
            </li>
            <li>
              4 people to supply lunch/water
            </li>
            <li>
              1-2 videographers
            </li>
          </ul>
          <p>
            Please
             
            <a
              href="https://signup.com/client/invitation2/secure/444699447581390086/false#/invitation"
              rel="noreferrer"
              target="_blank"
            >
              <b>
                click here
              </b>
            </a>
             
            to sign up to volunteer!
          </p>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      style={
        {
          "paddingInline": "20px",
        }
      }
    >
      <h4
        style={
          {
            "paddingBottom": "15px",
            "paddingTop": "22px",
            "textAlign": "center",
          }
        }
      >
        Progression Slideshow
      </h4>
      <div>
        <div
          className="slick-slider slick-initialized"
          dir="ltr"
        >
          <div
            className="slick-list"
            onClick={[Function]}
            onKeyDown={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={null}
            onMouseMove={null}
            onMouseUp={[Function]}
            onTouchCancel={null}
            onTouchEnd={[Function]}
            onTouchMove={null}
            onTouchStart={[Function]}
            style={{}}
          >
            <div
              className="slick-track"
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              style={
                {
                  "opacity": 1,
                }
              }
            >
              <div
                aria-hidden={false}
                className="slick-slide slick-active slick-current"
                data-index={0}
                onClick={[Function]}
                style={
                  {
                    "left": -0,
                    "opacity": 1,
                    "outline": "none",
                    "position": "relative",
                    "transition": "opacity 5000ms ease, visibility 5000ms ease",
                    "width": 0,
                  }
                }
                tabIndex="-1"
              >
                <div>
                  <div
                    style={
                      {
                        "display": "inline-block",
                        "width": "100%",
                      }
                    }
                    tabIndex={-1}
                  >
                    <img
                      alt="CLC Fellowship"
                      className="slide-show"
                      src="https://dl.dropboxusercontent.com/s/s1n4nxwc4pz3bml/CLC_Fellowship.png?dl=0"
                    />
                  </div>
                </div>
              </div>
              <div
                aria-hidden={true}
                className="slick-slide"
                data-index={1}
                onClick={[Function]}
                style={
                  {
                    "left": -0,
                    "opacity": 0,
                    "outline": "none",
                    "position": "relative",
                    "transition": "opacity 5000ms ease, visibility 5000ms ease",
                    "width": 0,
                  }
                }
                tabIndex="-1"
              >
                <div>
                  <div
                    style={
                      {
                        "display": "inline-block",
                        "width": "100%",
                      }
                    }
                    tabIndex={-1}
                  >
                    <img
                      alt="Psalm 127"
                      className="slide-show"
                      src="https://dl.dropboxusercontent.com/s/w4drwppxa9jrcmo/psalm127.jpg?dl=0"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <p
        style={
          {
            "textAlign": "center",
          }
        }
      >
        For progression videos, please checkout our
         
        <a
          href="https://www.youtube.com/channel/UCOra1rXiO-BHzMDNlLd9hFQ/videos"
          rel="noopener noreferrer"
          target="_blank"
        >
          YouTube channel
        </a>
        .
      </p>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      style={
        {
          "paddingInline": "20px",
        }
      }
    >
      <h4
        style={
          {
            "paddingBottom": "15px",
            "paddingTop": "22px",
            "textAlign": "center",
          }
        }
      >
        Construction Team
      </h4>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-item"
      >
        <div>
          <p>
            <strong>
              TEAM LEADERS:
            </strong>
          </p>
          <div>
            <div>
              <p>
                Matt Gart, Mike Preston, and Skip Zubrod
              </p>
              <strong>
                 RESPONSIBILITIES 
              </strong>
            </div>
            <ul>
              <li>
                Schedule workdays
              </li>
              <li>
                Schedule workers
              </li>
              <li>
                Coordinate with site supervisor
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      style={
        {
          "paddingInline": "20px",
        }
      }
    >
      <h4
        style={
          {
            "paddingBottom": "15px",
            "paddingTop": "22px",
            "textAlign": "center",
          }
        }
      >
        Support Team
      </h4>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-item"
      >
        <div>
          <p>
            <strong>
              TEAM LEADERS:
            </strong>
          </p>
          <div>
            <div>
              <p>
                Lisa Yagle, Kay Westermann, and Karen Zubrod
              </p>
              <p>
                <strong>
                  Sunday Updaters:
                </strong>
              </p>
              <p>
                Roger Holtman, Anita Ungs, and Paul Brummer
              </p>
              <p>
                <strong>
                  Publicity:
                </strong>
              </p>
              <p>
                Gary Lautenschlager
              </p>
              <p>
                <strong>
                  Web Page:
                </strong>
              </p>
              <p>
                Josh Sherman
              </p>
              <strong>
                 RESPONSIBILITIES 
              </strong>
            </div>
            <ul>
              <li>
                Make lunches
              </li>
              <li>
                Provide water
              </li>
              <li>
                Provide safety
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      style={
        {
          "paddingInline": "20px",
        }
      }
    >
      <h4
        style={
          {
            "paddingBottom": "15px",
            "paddingTop": "22px",
            "textAlign": "center",
          }
        }
      >
        Family Team
      </h4>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-item"
      >
        <div>
          <p>
            <strong>
              TEAM LEADERS:
            </strong>
          </p>
          <div>
            <div>
              <p>
                Bob and Lu Ann Yates, Ellen Holtman, and Debi Talin
              </p>
              <strong>
                 RESPONSIBILITIES 
              </strong>
            </div>
            <ul>
              <li>
                Connect with the family
              </li>
              <li>
                Provide move-in needs
              </li>
              <li>
                Introduce family to CLC
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    style={
      {
        "marginLeft": "-9px",
        "textAlign": "center",
      }
    }
  >
    <a
      href="http://www.elca.org/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="ELCA LOGO"
        id="elcaLogo"
        src="https://dl.dropboxusercontent.com/s/wkzubcmmm3pqst4/elca-logo.png?dl=0"
        style={
          {
            "margin": "auto",
            "marginLeft": "",
            "paddingTop": "30px",
            "width": "308px",
          }
        }
      />
    </a>
  </div>
</div>
`;

exports[`Habitat Project Renders the Habitat Project page 1`] = `
<div
  className="page-content"
>
  <div
    className="MuiGrid-root MuiGrid-container"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      style={
        {
          "marginInline": "0",
          "paddingInline": "20px",
        }
      }
    >
      <h3
        style={
          {
            "marginBottom": "0px",
            "paddingBottom": "16px",
            "paddingTop": "16px",
            "textAlign": "center",
          }
        }
      >
        CLC Habitat Project
      </h3>
      <p
        style={
          {
            "textAlign": "left",
          }
        }
      >
        College Lutheran Church has made a commitment of $50,000 to sponsor a home in the Roanoke Valley. This is the largest service project undertaken by the members of College Lutheran and we are delighted to be underway.
        <br />
        Here are some details:
        <ul>
          <li>
            This will be a rebuild project at 1004 Stewart Avenue, SE Roanoke.
          </li>
          <li>
            The existing structure will be gutted with hazardous material being mitigated before volunteers are on site.
          </li>
          <li>
            New construction will be added to the rear of the building.
          </li>
          <li>
            A family has yet to be confirmed for this home, but we expect a family will be designated in the coming weeks
          </li>
          <li>
            All level of construction skills will be needed. Workdays will probably be Wednesday and Saturday, but that may change. We typically work 8:00 â€“ 2:00, but half-time participation is welcome.
          </li>
          <li>
            Lunch, water, tools are all provided.
          </li>
        </ul>
      </p>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      style={
        {
          "paddingInline": "20px",
        }
      }
    >
      <h4
        style={
          {
            "paddingBottom": "15px",
            "paddingTop": "22px",
            "textAlign": "center",
          }
        }
      >
        Volunteer Sign Ups
      </h4>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-content-xs-center"
      >
        <div
          style={
            {
              "maxWidth": "500px",
            }
          }
        >
          <p>
            There will be a total of 42 days for volunteering to support the habitat build. For each workday we need to have the following volunteers:
          </p>
          <ul>
            <li>
              10 people for construction work
            </li>
            <li>
              4 people to supply lunch/water
            </li>
            <li>
              1-2 videographers
            </li>
          </ul>
          <p>
            Please
             
            <a
              href="https://signup.com/client/invitation2/secure/444699447581390086/false#/invitation"
              rel="noreferrer"
              target="_blank"
            >
              <b>
                click here
              </b>
            </a>
             
            to sign up to volunteer!
          </p>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      style={
        {
          "paddingInline": "20px",
        }
      }
    >
      <h4
        style={
          {
            "paddingBottom": "15px",
            "paddingTop": "22px",
            "textAlign": "center",
          }
        }
      >
        Progression Slideshow
      </h4>
      <div>
        <div
          className="slick-slider slick-initialized"
          dir="ltr"
        >
          <div
            className="slick-list"
            onClick={[Function]}
            onKeyDown={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={null}
            onMouseMove={null}
            onMouseUp={[Function]}
            onTouchCancel={null}
            onTouchEnd={[Function]}
            onTouchMove={null}
            onTouchStart={[Function]}
            style={{}}
          >
            <div
              className="slick-track"
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              style={
                {
                  "opacity": 1,
                }
              }
            >
              <div
                aria-hidden={false}
                className="slick-slide slick-active slick-current"
                data-index={0}
                onClick={[Function]}
                style={
                  {
                    "left": -0,
                    "opacity": 1,
                    "outline": "none",
                    "position": "relative",
                    "transition": "opacity 5000ms ease, visibility 5000ms ease",
                    "width": 0,
                  }
                }
                tabIndex="-1"
              >
                <div>
                  <div
                    style={
                      {
                        "display": "inline-block",
                        "width": "100%",
                      }
                    }
                    tabIndex={-1}
                  >
                    <img
                      alt="CLC Fellowship"
                      className="slide-show"
                      src="https://dl.dropboxusercontent.com/s/s1n4nxwc4pz3bml/CLC_Fellowship.png?dl=0"
                    />
                  </div>
                </div>
              </div>
              <div
                aria-hidden={true}
                className="slick-slide"
                data-index={1}
                onClick={[Function]}
                style={
                  {
                    "left": -0,
                    "opacity": 0,
                    "outline": "none",
                    "position": "relative",
                    "transition": "opacity 5000ms ease, visibility 5000ms ease",
                    "width": 0,
                  }
                }
                tabIndex="-1"
              >
                <div>
                  <div
                    style={
                      {
                        "display": "inline-block",
                        "width": "100%",
                      }
                    }
                    tabIndex={-1}
                  >
                    <img
                      alt="Psalm 127"
                      className="slide-show"
                      src="https://dl.dropboxusercontent.com/s/w4drwppxa9jrcmo/psalm127.jpg?dl=0"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <p
        style={
          {
            "textAlign": "center",
          }
        }
      >
        For progression videos, please checkout our
         
        <a
          href="https://www.youtube.com/channel/UCOra1rXiO-BHzMDNlLd9hFQ/videos"
          rel="noopener noreferrer"
          target="_blank"
        >
          YouTube channel
        </a>
        .
      </p>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      style={
        {
          "paddingInline": "20px",
        }
      }
    >
      <h4
        style={
          {
            "paddingBottom": "15px",
            "paddingTop": "22px",
            "textAlign": "center",
          }
        }
      >
        Construction Team
      </h4>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-item"
      >
        <div>
          <p>
            <strong>
              TEAM LEADERS:
            </strong>
          </p>
          <div>
            <div>
              <p>
                Matt Gart, Mike Preston, and Skip Zubrod
              </p>
              <strong>
                 RESPONSIBILITIES 
              </strong>
            </div>
            <ul>
              <li>
                Schedule workdays
              </li>
              <li>
                Schedule workers
              </li>
              <li>
                Coordinate with site supervisor
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      style={
        {
          "paddingInline": "20px",
        }
      }
    >
      <h4
        style={
          {
            "paddingBottom": "15px",
            "paddingTop": "22px",
            "textAlign": "center",
          }
        }
      >
        Support Team
      </h4>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-item"
      >
        <div>
          <p>
            <strong>
              TEAM LEADERS:
            </strong>
          </p>
          <div>
            <div>
              <p>
                Lisa Yagle, Kay Westermann, and Karen Zubrod
              </p>
              <p>
                <strong>
                  Sunday Updaters:
                </strong>
              </p>
              <p>
                Roger Holtman, Anita Ungs, and Paul Brummer
              </p>
              <p>
                <strong>
                  Publicity:
                </strong>
              </p>
              <p>
                Gary Lautenschlager
              </p>
              <p>
                <strong>
                  Web Page:
                </strong>
              </p>
              <p>
                Josh Sherman
              </p>
              <strong>
                 RESPONSIBILITIES 
              </strong>
            </div>
            <ul>
              <li>
                Make lunches
              </li>
              <li>
                Provide water
              </li>
              <li>
                Provide safety
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      style={
        {
          "paddingInline": "20px",
        }
      }
    >
      <h4
        style={
          {
            "paddingBottom": "15px",
            "paddingTop": "22px",
            "textAlign": "center",
          }
        }
      >
        Family Team
      </h4>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-item"
      >
        <div>
          <p>
            <strong>
              TEAM LEADERS:
            </strong>
          </p>
          <div>
            <div>
              <p>
                Bob and Lu Ann Yates, Ellen Holtman, and Debi Talin
              </p>
              <strong>
                 RESPONSIBILITIES 
              </strong>
            </div>
            <ul>
              <li>
                Connect with the family
              </li>
              <li>
                Provide move-in needs
              </li>
              <li>
                Introduce family to CLC
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    style={
      {
        "marginLeft": "-9px",
        "textAlign": "center",
      }
    }
  >
    <a
      href="http://www.elca.org/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="ELCA LOGO"
        id="elcaLogo"
        src="https://dl.dropboxusercontent.com/s/wkzubcmmm3pqst4/elca-logo.png?dl=0"
        style={
          {
            "margin": "auto",
            "marginLeft": "",
            "paddingTop": "30px",
            "width": "308px",
          }
        }
      />
    </a>
  </div>
</div>
`;
